### ISSUE TITLE
Installation issue

### ISSUE THREAD

**Issue by sheim** [popupcad/popupcad#11]
Hi, when running `python popupcad.py`, I get the following warnings:

    Warning: QApplication was created before pyqtgraph was imported; there may be problems (to avoid bugs, call QApplication.setGraphicsSystem("raster") before the QApplication is created).  
    Gazebo Plugin Not Found

The GUI starts, but many of the buttons are unclickable, or result in an error.

I running on OS X 10.10, and tried both with homebrew and anaconda, and get the same issues. Any ideas what the cause is?

**Comment by danaukes** [comment]
No, I'm traveling right now and without my Mac, so I can't debug the issue.
I can take a look at it when I return next week.

**Comment by danaukes** [comment]
(It looks to be a simple issue, probably due to an update.)
On Sep 30, 2015 2:22 AM, danaukes@gmail.com wrote:

No, I'm traveling right now and without my Mac, so I can't debug the issue.
I can take a look at it when I return next week.
On Sep 29, 2015 11:36 PM, "sheim" <notifications@github.com> wrote:

Hi, when running python popupcad.py, I get the following warnings:

Warning: QApplication was created before pyqtgraph was imported; there
may be problems (to avoid bugs, call
QApplication.setGraphicsSystem("raster") before the QApplication is
created).
Gazebo Plugin Not Found

The GUI starts, but many of the buttons are unclickable, or result in an
error.

I running on OS X 10.10, and tried both with homebrew and anaconda, and get
the same issues. Any ideas what the cause is?

—
Reply to this email directly or view it on GitHub
<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_popupcad_popupcad_issues_11&d=BQMCaQ&c=WO-RGvefibhHBZq3fL85hQ&r=yjnTLiBlKr4W189mDVx3e7kaEB8U4ZXO5uHigeERe7g&m=w7V9bUXEOxjNlyqTtRcVq1pSF1IWuCE0YP1PpUDIM_M&s=TBWllOza3WoU0TTNQ2vmsYTIp0rrJ0WEYxiFjvVimUw&e=>
.
